content/index.html

```diff
@@ -168,6 +168,10 @@ <h6>Join Us</h6>
               <a href="https://github.com/libp2p" target="_blank" class="btn-socials btn-github">GITHUB</a>
               <a href="https://twitter.com/libp2p" target="_blank" class="btn-socials btn-twitter">TWITTER</a>
             </div>
+            <div class="wrap">
+              <h6>Community Calendar</h6>
+              <p>If you're a libp2p implementer, consumer, or user you are welcome to join the community meetings we host regularly. Stay up to date by subscribing to the <a href="https://calendar.google.com/calendar/embed?src=libp2p.io_0q9682i3te7eanhe9q7ae1c58g%40group.calendar.google.com">libp2p Community Calendar</a> and joining community meetings such as triage sessions and community calls. Events can be added to the community calendar by inviting <a href="libp2p.io_0q9682i3te7eanhe9q7ae1c58g@group.calendar.google.com">the dedicated libp2p email.</a></p>
```

Comment:
```suggestion
              <p>If you're a libp2p implementer, consumer, or user you are welcome to join the community meetings we host regularly. Stay up to date by subscribing to the <a href="https://calendar.google.com/calendar/embed?src=libp2p.io_0q9682i3te7eanhe9q7ae1c58g%40group.calendar.google.com">libp2p Community Calendar</a> and joining community meetings such as triage sessions and community calls. Events can be added to the community calendar by inviting <a href="libp2p.io_0q9682i3te7eanhe9q7ae1c58g@group.calendar.google.com">the libp2p community calendar's email.</a></p>
```

Suggestion is to make it clear that this email address' only purpose is to receive calendar invites.